funding/generate_docx.py

```diff
@@ -0,0 +1,44 @@
+from docx import Document
+from io import BytesIO
+
+
+def create_funding_document(funding_body, title, pi_firstname, pi_lastname, pi_department, receiver, template):
```

Comment:
Does this document need multi-language support?

Reply:
Probably? We're supposed to accept correspondence in both English and Welsh, so if we're generating correspondence for someone then it seems to make sense. Not that we have translation strings yet.